### ISSUE TITLE
citation lists are misaligned

### ISSUE THREAD

**Issue by cissou8** [readthedocs/sphinx_rtd_theme#1411]
### Problem

Citation lists are not aligned.

### Reproducible Project

See for instance: https://siril.readthedocs.io/en/latest/processing/deconvolution.html#references  
Citations are inserted in the body of the document as `[paperref]_` and a list to gather them all at the end of the page as:
``` rst
.. [paperref] Authors, paper title, etc...

.. [paperref2] Authors, paper title, etc...
```

#### Error Logs/Results

As rendered now:
sphinx 6.0.0, rtd-theme 1.2.0rc2
![image](https://user-images.githubusercontent.com/65344766/212420112-5d619af8-3d07-40c7-853b-00949bfb3124.png)

#### Expected Results

As rendered with sphinx 5.3.0, rtd-theme 1.1.0
![image](https://user-images.githubusercontent.com/65344766/212420285-87fdae6d-84bf-4c48-bc55-ec43795d7fb7.png)

Do not hesitate if you want me to test further,

Thx,

C.

**Comment by benjaoming** [comment]
Thanks for posting this issue. It's definitely valid, so let's keep it around to track it and identify a solution.

The problem was introduced because of docutils 0.18 which is the lowest supported version in Sphinx 6. Basically, the new DOM structure that is created by docutils 0.18 is very challenging to support with a nice table as the former.

You can see how many hard efforts unfolded here: #1381

**Comment by cissou8** [comment]
Indeed, didn't search thoroughly enough throughout the tickets! Think I searched only the opened ones.  
Sorry I can't help much with CSS/html as most of it feels like pure magic to me, but happy to test whenever needed!

**Comment by benjaoming** [comment]
@cissou8 I have to say that the CSS required for displaying changes in docutils 0.18 are quite advanced and the work done by @agjohnson is already quite substantial. I'm not sure if there is a quick way to fix this.
